HospitalManagement/src/main/java/com/realhospitalmanagementsystem/entity/Patient.java

```diff
@@ -0,0 +1,30 @@
+package com.realhospitalmanagementsystem.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Patient {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String patientName;
+    private int age;
+    private String gender;
+    private String address;
+    private String mobileNo;
+    private String department;
```

Comment:
Add more field like doctor prescription and appointment number i want to search patient based on appointment number